content/docs/nav.yml

```
@@ -32,7 +32,7 @@
     - id: lifting-state-up
       title: Lifting State Up
     - id: composition-vs-inheritance
-      title: Composition vs Inheritance
+      title: Composition et Héritage
```

```suggestion
      title: Composition ou héritage ?
```

À part ça, GG d'avoir pensé à la nav 💯